src/Compilers/CSharp/Test/Symbol/Symbols/ModuleInitializersTests.Ignored.cs

```diff
@@ -0,0 +1,289 @@
+using System.Runtime.CompilerServices;
+
+class C
+{
+    [ModuleInitializer]
+    public int P { get; set; }
+}
+
+namespace System.Runtime.CompilerServices { class ModuleInitializerAttribute : System.Attribute { } }
+";
+            var verifier = CompileAndVerify(source, parseOptions: s_parseOptions);
+
+            verifier.VerifyMemberInIL("<Module>..cctor", expected: false);
+        }
+
+        [Fact]
+        public void IgnoredOnIndexer()
+        {
+            string source = @"
+using System.Runtime.CompilerServices;
+
+class C
+{
+    [ModuleInitializer]
+    public int this[int p] => p;
+}
+
+namespace System.Runtime.CompilerServices { class ModuleInitializerAttribute : System.Attribute { } }
+";
+            var verifier = CompileAndVerify(source, parseOptions: s_parseOptions);
+
+            verifier.VerifyMemberInIL("<Module>..cctor", expected: false);
+        }
+
+        [Fact]
+        public void IgnoredOnField()
+        {
+            string source = @"
+using System.Runtime.CompilerServices;
+
+class C
+{
+    [ModuleInitializer]
+    public int F;
+}
+
+namespace System.Runtime.CompilerServices { class ModuleInitializerAttribute : System.Attribute { } }
+";
+            var verifier = CompileAndVerify(source, parseOptions: s_parseOptions);
+
+            verifier.VerifyMemberInIL("<Module>..cctor", expected: false);
+        }
+
+        [Fact]
+        public void IgnoredOnModule()
+        {
+            string source = @"
+using System.Runtime.CompilerServices;
+
+[module: ModuleInitializer]
+
+namespace System.Runtime.CompilerServices { class ModuleInitializerAttribute : System.Attribute { } }
+";
+            var verifier = CompileAndVerify(source, parseOptions: s_parseOptions);
+
+            verifier.VerifyMemberInIL("<Module>..cctor", expected: false);
+        }
+
+        [Fact]
+        public void IgnoredOnAssembly()
+        {
+            string source = @"
+using System.Runtime.CompilerServices;
+
+[assembly: ModuleInitializer]
+
+namespace System.Runtime.CompilerServices { class ModuleInitializerAttribute : System.Attribute { } }
+";
+            var verifier = CompileAndVerify(source, parseOptions: s_parseOptions);
+
+            verifier.VerifyMemberInIL("<Module>..cctor", expected: false);
+        }
+
+        [Fact]
+        public void IgnoredWhenConstructorArgumentIsSpecified()
+        {
+            string source = @"
+using System.Runtime.CompilerServices;
+
+class C
+{
+    [ModuleInitializer(42)]
+    internal static void M()
+    {
+    }
+}
+
+namespace System.Runtime.CompilerServices
+{
+    class ModuleInitializerAttribute : System.Attribute
+    {
+        public ModuleInitializerAttribute(int p) { }
+    }
+}
+";
+            var verifier = CompileAndVerify(source, parseOptions: s_parseOptions);
+
+            verifier.VerifyMemberInIL("<Module>..cctor", expected: false);
+        }
+
+        [Fact]
+        public void IgnoredWhenNamedArgumentIsSpecified()
```

Comment:
> Wait, if this test is passing then it means that either the test methodology is flawed and all these tests are broken or that the attribute decoder is actually not matching when a named argument is specified. I think it's the former because I had to change the test methodology in the previous PR but forgot to update this one.

I think this is consistent with how we deal with attributes in source today. It looks like we ignore extra properties. 

---
In reply to: [417362650](https://github.com/dotnet/roslyn/pull/43301#discussion_r417362650) [](ancestors = 417362650)